specifications/active/aRMT-1.1.0.yml

```diff
@@ -16,7 +32,7 @@ data:
     questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/esm/esm_armt.json
   - type: AUDIO
     topic: questionnaire_audio
-    value_schema: .active.opensmile.OpenSmile2AudioRecording
+    value_schema: .active.opensmile.OpenSmile2AudioAnalysis
```

Comment:
@kerzmaximilian suggested that he will be posting a base64 encoded string and not a raw audio file in bytes. We added the reciteText option in the audio analysis schema as well.

Reply:
In principle that's fine, although I'm not exactly sure why, but if the contents are going to be a audio recording, you could change the schema of audio recording data to contain base64 encoded data instead of bytes... Semantically, it is an audio recording though, right?

Reply:
yes true. no analysis is done on the recording. I will update the schema